tests/PHPUnit/Integration/Settings/SystemSettingTest.php

```diff
@@ -91,6 +92,7 @@ public function test_getSettingValue_shouldThrowException_IfUserHasNotEnoughPerm
      */
     public function test_getSettingValue_shouldThrowException_IfAnonymousTriedToReadValue()
     {
+        $this->setAnoymous();
```

Comment:
> Anoymous? Shouldn't it be Anonymous?

Good point. You can only write it once and that sends the message to the developer :)